.github/workflows/dockerpublish.yml

```diff
@@ -40,7 +40,11 @@
 #   with:
 #     sus: 'mkdir -p ./hits/postgresql/data'
 #     service_name: 'hits'
+  Health-checks-homeassistant:
+    uses: 'tomMoulard/make-my-server/.github/workflows/healthcheck.workflow.tmpl.yml@master'
+    with:
+      service_name: 'homeassistant'
```

Comment:
**Suggestion:** Consider using a specific version of the workflow template instead of the master branch. This will ensure that your workflow doesn't break if the template is updated in a way that's incompatible with your usage.
```suggestion
  Health-checks-homeassistant:
    uses: 'tomMoulard/make-my-server/.github/workflows/healthcheck.workflow.tmpl.yml@v1.0.0'
    with:
      service_name: 'homeassistant'
```